config/logstash.yml

```diff
@@ -50,6 +50,15 @@
 #
 # pipeline.unsafe_shutdown: false
 #
+# Defaults to false. Setting this to true will rescue all unexpected errors in the
+# pipeline. Logstash will do all of the following: drop the erroring event,
+# log an error-level message, and continue processing.
+# If set to false logstash will immediately exit with an error in these
+# conditions, which may be more safe since potentially fewer events would be dropped.
+#
+# pipeline.rescue_all
```

Comment:
I think `rescue_all` is the wrong term. It's much less about rescuing (events) than it is about resovling error-causing event from the pipeline by destroying the event.